src/api/src/model.php

```diff
@@ -55,21 +55,21 @@ public static function getValue($pointID, $timestamp){
 	}
 	public static function getValuesByBuildingInRange($buildingID, $start, $end){
 		$db = Database::getInstance();
-		$sth = $db->prepare("SELECT * FROM PointValues 
-			LEFT JOIN Points ON PointValues.PointID=Points.ID
+		$sth = $db->prepare("SELECT PointValues.*, PointTypes.* FROM PointValues 
+			LEFT JOIN Points ON PointValues.PointID=Points.ID 
 			LEFT JOIN Rooms ON Rooms.ID = Points.RoomID 
 			LEFT JOIN PointTypes ON PointTypes.ID=Points.PointTypeID 
 			WHERE Rooms.BuildingID=? AND PointTimestamp>? and PointTimestamp < ?");
 		$sth->execute([$buildingID, $start, $end]);
-		return $sth->fetch();
+		return $sth->fetchAll(PDO::FETCH_ASSOC);
```

Comment:
We run into memory issues if we try and get all of a buildings point values for any kind of significant interval (in excess of like 3/4 days). May need to raise this limit if we need functionality like that, should just be a setting in php.ini that we need to get mike tie to update